src/options/options.js

```diff
@@ -103,6 +103,19 @@ async function setupListeners() {
   if (currentValue) {
     document.querySelector(`.panel[data-value="${currentValue}"]`).classList.add("selected");
   }
+
+  document.getElementById("notificationActive").addEventListener("change", (event) => {
```

Comment:
You can simply do this here and ignore the fact that the event listener isn't normally an async function:

```javascript
document.getElementById("notificationActive").addEventListener("change", async (event) => {
	...
});
```

Reply:
ok, I'll do it.